<launch>
	<include file="$(find realsense2_camera)/launch/rs_aligned_depth.launch">
		<arg name="camera" value="cam_L"/>
		<arg name="serial_no" value="332322072918"/>
		<arg name="filters" value="pointcloud"/>
		<arg name="enable_pointcloud" value="true"/>
	</include>
	<!-- <include file="$(find realsense2_camera)/launch/rs_aligned_depth.launch">
		<arg name="camera" value="cam_M"/>
		<arg name="serial_no" value="938422071315"/>
		<arg name="filters" value="pointcloud"/>
		<arg name="enable_pointcloud" value="true"/>
	</include> -->
	<include file="$(find realsense2_camera)/launch/rs_aligned_depth.launch">
		<arg name="camera" value="cam_R"/>
		<arg name="serial_no" value="327122075069"/>
		<arg name="filters" value="pointcloud"/>
		<arg name="enable_pointcloud" value="true"/>
	</include>
</launch>
